bedrock/firefox/views.py

```diff
@@ -660,120 +660,6 @@ def ios_testflight(request):
                              {'newsletter_form': newsletter_form})
 
 
-def features(request):
-    if switch('firefox-57-release'):
-        template = 'firefox/features/quantum/index.html'
-    else:
-        template = 'firefox/features/index.html'
-
-    return l10n_utils.render(request, template)
-
-
-class FeaturesPrivateBrowsingView(BlogPostsView):
```

Comment:
This is removing the blog posts from feature pages that are currently visible on prod. Is that intended?

Reply:
Oh, nope. I got a little carried away cleaning up `views.py`. 😳 Restored views where needed.